src/main/java/com/deavensoft/paketomat/courier/CourierServiceImpl.java

```diff
@@ -22,38 +22,39 @@ public class CourierServiceImpl implements CourierService {
     private final CenterService centerService;
     private final CityService cityService;
     private final DispatcherService dispatcherService;
-    private List<Package> packagesToDispatch = new ArrayList<>();
-    private List<City> citiesToDispatch = new ArrayList<>();
+    private final List<Package> packagesToDispatch = new ArrayList<>();
+    private final List<City> citiesToDispatch = new ArrayList<>();
 
-    public List<Courier> findAllCouriers(){
+    public List<Courier> findAllCouriers() {
         return courierRepository.findAll();
     }
 
-    public void saveCourier(Courier newCourier){
+    public void saveCourier(Courier newCourier) {
         courierRepository.save(newCourier);
     }
 
 
-    public Optional<Courier> getCourierById(Long id){
+    public Optional<Courier> getCourierById(Long id) {
         return courierRepository.findById(id);
     }
 
-    public void deleteCourierById(Long id){
+    public void deleteCourierById(Long id) {
         courierRepository.deleteById(id);
     }
+
     @Override
     public List<Package> getPackagesForCourier(String city) throws PaketomatException {
         getPackagesToDispatch();
```

Comment:
According to the name getPackagesToDispatche, I expect that I'll get a collection of items. But I noticed that the method has a void as return type. Please, fix it. The method should return List<Package>. 

- [ ] Fix it.

Reply:
Done.

Reply:
Did you test changes?

Reply:
getPackagesToDispatch() and findCitiesInRadius(city) return lists, please use these lists as arguments for filterPackagesToDispatch.

Reply:
Fixed.

Reply:
Solution 1: 

`   @Override
    public List<Package> getPackagesForCourier(String city) throws PaketomatException {
        final List<Package> packagesToDispatch = getPackagesToDispatch();
        final List<City> citiesInRadius = findCitiesInRadius(city);
        return deliverPackageInPaketomat(filterPackagesToDispatch(packagesToDispatch,citiesInRadius));
    }`


Solution 2: 
`@Override
    public List<Package> getPackagesForCourier(String city) throws PaketomatException {
        return deliverPackageInPaketomat(filterPackagesToDispatch(getPackagesToDispatch(),findCitiesInRadius(city)));
    }`

Reply:
Solution 1

Reply:
Solution 1? 😄